### PR TITLE
He creado una rama nueva desde LoginUI para hacer el commit y el push

### PR DESCRIPTION
**Pull request by evivar**
request del commandLogin bien hecha

### CODE REVIEW

File: HostAbroad/src/main/java/com/presentation/commands/CommandLogin.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import com.business.ASFactory.ASFactory;
+
+public class CommandLogin extends Command {
+
+	@Override
+	public Pair<Integer, Object> execute(Object transfer) {
+		int result;
+		ASUser saUser = ASFactory.getInstance().createASUser();
+		boolean loged = saUser.loginUser();
```
Comment (GasanNazer) [inline HostAbroad/src/main/java/com/presentation/commands/CommandLogin.java]: You have to pass the transfer. How the AS is going to check if the user have entered the right email and pass_

File: HostAbroad/src/main/java/com/presentation/commands/CommandLogin.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		result = loged ? 0 : 1;
```
Comment (GasanNazer) [inline HostAbroad/src/main/java/com/presentation/commands/CommandLogin.java]: we are using reverted logic pls change it

Comment (SirTorresDev) [inline HostAbroad/src/main/java/com/presentation/commands/CommandLogin.java]: I think that you should be return 1 when loged and 0 when has not loged

**Review by SirTorresDev — COMMENTED** [review]
Modify the code, and lauch other pull request

**Pull request merged**